src/databricks/labs/ucx/source_code/notebooks/cells.py

```diff
@@ -171,7 +172,27 @@ def build_dependency_graph(self, parent: DependencyGraph, problem_collector: Cal
             end_line=start_line,
             end_col=len(self._original_code),
         )
-        problem_collector(problem)
+        return MaybeGraph(None, [problem])
+
+    def _adjust_import_problems(
```

Comment:
this is a %run cell, _adjust_import_problems is misleading

Reply:
renamed to `_adjust_problem_source_paths_and_line_numbers`